Source/WebCore/animation/ViewTimeline.cpp

```diff
@@ -294,16 +297,18 @@ ScrollTimeline::Data ViewTimeline::computeTimelineData(const TimelineRange& rang
     return { currentScrollOffset, rangeStart + ScrollTimeline::floatValueForOffset(range.start.offset, rangeEnd - rangeStart), rangeEnd - ScrollTimeline::floatValueForOffset(range.end.offset, rangeEnd - rangeStart) };
 }
 
-const CSSNumericValue& ViewTimeline::startOffset() const
+const CSSNumericValue& ViewTimeline::startOffset()
 {
     auto data = computeTimelineData();
-    return CSSNumericFactory::px(data.rangeStart);
+    m_startOffset = CSSNumericFactory::px(data.rangeStart);
+    return m_startOffset;
```

Comment:
I fail to understand the point of introducing a new instance variable since we won't be using that value ever. What is the benefit of that change?

Reply:
It's a bit confusing but we need to return a reference to a `CSSNumericValue` since it's an abstract class. We need the member variable to hold the value which we return as a reference. Not sure if there's a more clever way to do this.

Reply:
Maybe change the method signature to not be a reference?

Reply:
Also, why did this work prior to this change?

Reply:
We can't since `CSSNumericValue` is an abstract class (no copy constructor). Not entirely sure how this worked previously.